Lib/multiprocessing/resource_sharer.py

```diff
@@ -113,10 +112,7 @@ def _afterfork(self):
         for key, (send, close) in self._cache.items():
             close()
         self._cache.clear()
-        # If self._lock was locked at the time of the fork, it may be broken
-        # -- see issue 6721.  Replace it without letting it be gc'ed.
-        self._old_locks.append(self._lock)
-        self._lock = threading.Lock()
```

Comment:
It is really nice to see this hack being removed :-) Well, technically, _at_fork_reinit() basically does the same. But it's now hidden and has a single implementation: _at_fork_reinit() method ;-)